.github/workflows/publish_docs.yaml

```diff
@@ -0,0 +1,20 @@
+name: Public Docs
```

Comment:
Move this file to the [operator workflows repository](https://github.com/canonical/operator-workflows) and refactor the existing ones if needed to resuse it

Reply:
It turns out the publish charm already includes publish docs as a part of it.
This is file is unneeded.